Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -2077,7 +2261,7 @@
 					"HC_LINK_STATIC=1",
 					"HC_TRACE_BUILD_LEVEL=HC_PRIVATE_TRACE_LEVEL_IMPORTANT",
 				);
-				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
```

Comment:
Should this still be NO?

Reply:
Missed that one, I changed all of them to be YES now